### ISSUE TITLE
Wow! Amazing addon!!

### ISSUE THREAD

**Issue by RealDavidoff** [pinkienort/AutoScrolling#116]
Hisakazu I am only here to say: What a fantastic addon you made here!
I don't know why some people complain, they probably haven't seen your elaborate settings page ;-)
It's SO EASY to use, even I could use it out of the box! :-)

(sorry, couldn't find how to send you an email, this page seems most likely you see it)

I am wondering: Can your AutoScrolling addon somehow(?) be used with seamonkey browser?
I checked but you seem not to have uploaded it to the repository?
Any way that I manually install it on that browser please?

(Reason I'd LOVE this for seamonkey too: It's the only browser that works with "Pin4Ever" social tool and I'd love to combine it with your Autoscrolling addon for a Winautomation job that I built ;-)
Basically I'd use your Autoscrolling to slowly auto-scroll the long pages while copying out the DOM with my automation.
- The DOM changes upon scrolling when the website executes their javascripts, you know that. This is how they display those loooong photo pages.... LOL

I am NOT a programmer, no. But if your Autoscrolling addon can somehow be added to seamonkey too, this would be awesome! :-)
